src/pass/detect_device.cc

```diff
@@ -0,0 +1,30 @@
+/*!
+ *  Copyright (c) 2018 by Contributors
+ * \file detect_device.cc
+ */
+
+#include <tvm/ir_pass.h>
+#include <tvm/ir_mutator.h>
+#include "../pass/ir_util.h"
+
+namespace tvm {
+namespace ir {
+
+class DetectDevice : public IRMutator {
+ public:
+  DetectDevice() {}
+  Stmt Detect(Stmt stmt) {
+    Stmt body = AttrStmt::make(make_zero(Int(32)),
```

Comment:
Seems that this does not do anything except for the wrapping, is it intentional? The simple wrapping can be constructed by directly use AttrStmt constructor and does not seems to need a pass